### ISSUE TITLE
Argument list too long and hidden validation error

### ISSUE THREAD

**Issue by tomschr** [tomschr/daps#679]
### Problem description

@jfaltenbacher  reported that issue in our [internal Slack channel](https://suse.slack.com/archives/C02DR0BQ9C0/p1650523748278989). The daps command fails with the following error message:

```
$ daps -v -d DC-opensuse-gnome-user validate 
make: execvp: /bin/bash: Argument list too long
make: *** [/usr/share/daps/make/validate.mk:68: validate] Error 127
```

After further examination, it was a validation error. However, this error message was hidden behind the above bash line.

The first lines of the real error message looks like:

```
if [[ -n ".../SUSE/doc-sle/build/.profiled/x86_64_osuse/deployment_troubleshooting.xml:206:69: error: attribute "format" not allowed here; expected attribute "align", "annotations", ...
[... overly long line pruned ...]
```

@fsundermeyer Could you have a look?

### Expected behavior

daps should report the validation error, not the bash line from above.

### Steps to reproduce problem

1. Preparation of GeekoDoc v2.1.0:
    a. Clone the Geekodoc repository with `git clone git@github.com:openSUSE/geekodoc.git`
    b. Checkout tag v2.1.0: `git checkout v2.1.0`
    c. Install the build requirements: `sudo zypper install python3-rnginline trang`
    d. Build the Geekodoc schema with: `./build.sh`. The schema can be found at `dist/geekodoc/rng/2_5.2/geekodoc-v2-flat.rnc`.
   e. Convert RNC to RNG: `trang dist/geekodoc/rng/2_5.2/geekodoc-v2-flat.rnc dist/geekodoc/rng/2_5.2/geekodoc-v2-flat.rng`
2. Edit `~/.config/daps/dapsrc` and search for the `DOCBOOK5_RNG_URI` line. Add the path to the **RNG**(!) file and prefix it with `file://`.
3. Build the openSUSE Gnome User guide as mentioned above.

The funny thing is, you only get the full error message if you use `-vvv` or `--debug`.

**Comment by fsundermeyer** [comment]
Checks in validate.mk may fail if the error message that is captured in $FAULTY_... contains certain unescaped characters:

```
 if [[ -n "/local/git/doc-sle/build/.profiled/x86_64_zseries_power_aarch64_sles/deployment_yast_installer.xml:110:31: error: value of attribute "os" is invalid; must be a string matching the regular expression "([\-\._0-9a-zA-Z]+;)*([\-\._0-9a-zA-Z]+)" /local/git/doc-sle/build/.profiled/x86_64_zseries_power_aarch64_sles/libvirt_guest_installation.xml:419:31: error: value of attribute "os" is invalid; must be a string matching the regular expression "([\-\._0-9a-zA-Z]+;)*([\-\._0-9a-zA-Z]+)"" ]]; then \
  ccecho "error" "Fatal error: The document contains XML errors:"; \
  echo -e "/local/git/doc-sle/build/.profiled/x86_64_zseries_power_aarch64_sles/deployment_yast_installer.xml:110:31: error: value of attribute "os" is invalid; must be a string matching the regular expression "([\-\._0-9a-zA-Z]+;)*([\-\._0-9a-zA-Z]+)" /local/git/doc-sle/build/.profiled/x86_64_zseries_power_aarch64_sles/libvirt_guest_installation.xml:419:31: error: value of attribute "os" is invalid; must be a string matching the regular expression "([\-\._0-9a-zA-Z]+;)*([\-\._0-9a-zA-Z]+)""; \
  echo "--------------------------------"; \
fi
/bin/bash: -c: line 1: syntax error in conditional expression: unexpected token `('
```

Depending on the length of the error message it either fals as stated in the original report or is silently ignored.

**Comment by fsundermeyer** [comment]
Fixed with 5dcf5b6
